H/HarfBuzz/build_tarballs.jl

```diff
@@ -39,13 +39,14 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    "Graphite2_jll",
-    "Libffi_jll",
-    "Gettext_jll",
-    "Glib_jll",
-    "FreeType2_jll",
-    "Fontconfig_jll",
-    "Cairo_jll",
+    Dependency("Glib_jll"),
+    Dependency("FreeType2_jll"),
+    Dependency("Graphite2_jll"),
+    Dependency("Libffi_jll"),
+    Dependency("Gettext_jll"),
```

Comment:
Just speaking out loud.  I've never verified my hypothesis, but I think that also `Gettext_jll` is usually a build-only dependency.  We can leave as it is now, but I'd like to revise this at a later point.

Reply:
For example, none of the products of Harfbuzz is linked against libgettextlib.  I don't know if other files from this package are used though

Reply:
Yeah I was trying to determine that and I was looking at the Archlinux list and the Gentoo lists and didn't come to a conclusive answer.